frontends/web/src/api/backend.ts

```diff
@@ -49,3 +49,26 @@ export const renameAccount = (accountCode: string, name: string): Promise<ISucce
 export const reinitializeAccounts = (): Promise<null> => {
     return apiPost('accounts/reinitialize');
 };
+
+export interface AoppAccount {
+    name: string;
+    code: AccountCode;
+}
+
+export interface Aopp {
+    // See backend/aopp.go for a description of the states.
+    state: 'error' | 'inactive' | 'awaiting-keystore' | 'choosing-account' | 'syncing' | 'signing' | 'success';
+    accounts: AoppAccount[];
+    // See backend/errors.go for a description of the errors.
+    errorCode: '' | 'aoppUnsupportedAsset' | 'aoppVersion' | 'aoppInvalidRequest' | 'aoppNoAccounts' | 'aoppUnsupportedKeystore' | 'aoppUnknown' | 'aoppSigningAborted' | 'aoppCallback';
+    address: string;
+    callbackHost: string;
+}
+
+export const aoppCancel = (): Promise<null> => {
+    return apiPost('aopp/cancel');
+};
+
+export const aoppChooseAccount = (accountCode: AccountCode): Promise<null> => {
+    return apiPost('aopp/choose-account', { accountCode });
+};
```

Comment:
I'd move this to `api/aopp.ts` but can do later

Reply:
Done